asyncpg/connect_utils.py

```diff
@@ -383,6 +384,11 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
                 if target_session_attrs is None:
                     target_session_attrs = dsn_target_session_attrs
 
+            if 'krbsrvname' in query:
+                val = query.pop('krbsrvname')
+                if krbsrvname is None:
+                    krbsrvname = val
```

Comment:
Please add handling of the `PGKRBSRVNAME` environment variable as well to stay compatible with libpq.

Reply:
Done.